sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/OpenAIAsyncClient.java

```diff
@@ -310,6 +401,30 @@ public Mono<Completions> getCompletions(String deploymentId, CompletionsOptions
                 .map(protocolMethodData -> protocolMethodData.toObject(Completions.class));
     }
 
+    /**
+     * Gets completions for the provided input prompts. Completions support a wide variety of tasks and generate text
+     * that continues from or "completes" provided prompt data.
+     *
+     * @param deploymentId deployment id of the deployed model.
+     * @param prompt The prompts to generate values from.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
+     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
+     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return completions for the provided input prompts. Completions support a wide variety of tasks and generate text
+     *     that continues from or "completes" provided prompt data on successful completion of {@link Mono}.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<Completions> getCompletions(String deploymentId, String prompt) {
+        RequestOptions requestOptions = new RequestOptions();
+        CompletionsOptions completionsOptions = CompletionsUtils.DefaultCompletionsOptions(prompt);
+        return getCompletionsWithResponse(deploymentId, BinaryData.fromObject(completionsOptions), requestOptions)
```

Comment:
Call the convenience method instead `getCompletions(deploymentId, completionOptions)`, so you don't have to repeat the code to convert BinaryData into Completions.